src/config/serverConfigDefaults.ts

```diff
@@ -41,6 +41,7 @@ const ServerConfigDefaults: Partial<IServerConfig> = {
     show_civic: false,
     show_pathway_mapper: true,
     show_mutation_mapper_tool_grch38: true,
+    show_transcript_dropdown: true,
```

Comment:
@leexgh wonder if we should make it false for now so we can test in production before making true the default for everyone

Reply:
it is convenient with testing tho for now, but might make sense to change before merging